src/components/Icons/index.tsx

```diff
@@ -48,6 +48,8 @@ export const StepForward = <img src={ClockwiseArrow} />;
 export const StepBack = <img src={CounterClockwiseArrow} />;
 export const BetaTag = <img src={Beta} style={{ width: "42px" }} />;
 export const UpRightArrow = <ArrowLeftOutlined rotate={135} />;
+export const OrthographicCamera = <img src={Orthographic} />;
+export const PerspectiveCamera = <img src={Perspective} />;
```

Comment:
Somehow (probably during merge conflict / resolution), these lines are missing in the autoconversion feature branch, even though they've been merged into main. I added them back in here, but they have nothing to do with our overall change set